pkg/operator/staticpod/README.md

```diff
@@ -0,0 +1,161 @@
+# Static Pod Controllers
+
+OpenShift 4.x deploys the following foundational components as [static
+pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/):
+
+- etcd
+- kube-apiserver
+- kube-controller-manager
+- kube-scheduler
+
+These components comprise the minimum control plane on which all other
+OpenShift components are hosted.
```

Comment:
The are created a static pods, so that when a master reboots, the kubelet will immediately attempt to start these pods even in the absence of a connection to a kube-apiserver.  This capability can be valuable for any component which must logically be available *before* the kube-control-plane is fully available.  Another logical candidate is the machine-api-approver to approver CSRs